gorapass/urls.py

```diff
@@ -7,13 +7,16 @@
     path('hikes', views.hikes, name='hikes'),
     path('stamps', views.stamps, name='stamps'),
     path('stamps/<int:stamp_id>', views.stamp, name='stamp'),
+    path('stamps/<int:stamp_id>/get_status', views.get_stamp_status, name='get_stamp_status'),
     path('hikes/<int:hike_id>', views.hike, name='hike'),
     path('users/<int:user_id>', views.user, name='user'),
     path('users/completed_hikes', views.user_completed_hikes, name='user_completed_hikes'),
     path('users/<int:user_id>/completed_hikes/add', views.add_completed_hike, name='add_completed_hike'),
     path('users/<int:user_id>/completed_hikes/delete', views.delete_completed_hike, name='delete_completed_hike'),
+    path('users/open_stamps', views.user_open_stamps, name='user_open_stamps'),
```

Comment:
For the front end, I like the idea of using 'open' instead of incomplete, but I think since we have 'completed' on the backend it would be best to use `completed`/`incomplete` stamps.